frontend/src/scenes/session-recordings/components/AiFilter/AiFilterThread.tsx

```diff
@@ -0,0 +1,80 @@
+import { ProfilePicture, Spinner, Tooltip } from '@posthog/lemon-ui'
+import { BuiltLogic, useActions, useValues } from 'kea'
+import { LemonMarkdown } from 'lib/lemon-ui/LemonMarkdown'
+import { sessionRecordingsPlaylistLogicType } from 'scenes/session-recordings/playlist/sessionRecordingsPlaylistLogicType'
+import { userLogic } from 'scenes/userLogic'
+
+import { aiFilterLogic } from './aiFilterLogic'
+
+export function AiFilterThread({ logic }: { logic: BuiltLogic<sessionRecordingsPlaylistLogicType> }): JSX.Element {
```

Comment:
we tend not to pass logics around
you can pass the props down

or you can wrap the lot in 

```
<BindLogic logic={myLogic} props={theProps}>
 {theChildren}
</BindLogic>
```

now in children you can do

`const logicInstance = myLogic()`

and you don't need to know the props
this is clever (but also scares me sometimes since knowing if a logic is bound can get messy)